packages/modular-scripts/src/typecheck.ts

```diff
@@ -0,0 +1,65 @@
+import isCI from 'is-ci';
+import path from 'path';
+import ts from 'typescript';
+import chalk from 'chalk';
+import getPackageMetadata from './buildPackage/getPackageMetadata';
+import { reportTSDiagnostics } from './buildPackage/reportTSDiagnostics';
```

Comment:
Can we move this to `utils` if we're going to re-use it?

Reply:
I'm going to use ts.formatDiagnostic to handle the parsing errors instead